package.json

```diff
@@ -29,7 +29,9 @@
   "dependencies": {
     "@actions/core": "^1.2.4",
     "@actions/github": "^4.0.0",
-    "got": "^11.5.0"
+    "act": "^0.0.6",
```

Comment:
Can we remove `act` and `ncc`?  The right `ncc` one is `@vercel/ncc` that is in devDependencies and the `act` tool has to be installed separately and is not the same as the `act` on npm. Thank you for adding the Windows instructions. I am sure that adding these here was part of trying to get it to work on Windows.

Reply:
I didn't install act on my machine. It seemed slightly OTT for what I wanted to achieve because it relies on docker being installed. I'll just remove it, but I won't be able to test the change.

Reply:
We have some integration type tests that use act. No big deal, if you didn't install. The normal behavior tests don't require it. Thanks!